frontend/app/src/composables/filter-paginate.ts

```diff
@@ -202,7 +204,29 @@ export const usePaginationFilters = <
     {
       immediate: false,
       resetOnExecute: false,
-      delay: 0
+      delay: 0,
+      onError(e) {
+        const { notify } = useNotificationsStore();
+        const { t } = useI18n();
+        const error = e as AxiosError<{ message: string }>;
+        const path = error.config?.url;
+        let { message, code } = error;
+
+        if (error.response) {
+          message = error.response.data.message;
+          code = error.response.status.toString();
+        }
+
+        logger.error(error);
+        if (code === '500') {
```

Comment:
Maybe show the error on any 400/500 error?

In general the api should not fail but if it returns any 400/500 failure we should notify the user.

Reply:
okay, will do